src/blockly/generators/propc/variables.js

```diff
@@ -229,9 +232,9 @@ Blockly.Blocks.array_get = {
 
         // Init the array name to a default if there are no existing
         // array variable blocks
-        if (v_check) {
+        //if (v_check) {
```

Comment:
Same issue here, where commented out code should be removed. If we're not sure about it, the code shouldn't be in the PR.